src/main/java/lesson03/conditions/BiggerRectangle.java

```diff
@@ -0,0 +1,37 @@
+package lesson03.conditions;
+/*
+Определить, какой прямоугольник больше по площади. Если больше первый прямоугольник, то результат 1, иначе 2.
+Ограничения: widht1, height1, width2, height2 > 0.
+int biggerRectange(int widht1, int height1, int width2, int height2)
+Входные аргументы  int widht1, int height1, int width2, int height2
+Тип результата int
+*/
+
+import utils.errorhandling.ExceptionHandling;
+
+public class BiggerRectangle {
+    public static void main (String[] args) {
+        System.out.println(biggerRectangle(110,20,30,40));
+    }
+
+    public static int biggerRectangle (int width1, int height1, int width2, int height2){
+        int numberOfBiggerRectangle = 0;
+        int squareOfFirstRectangle = width1 * height1;
+        int squareOfSecondRectangle = width2 * height2;
+        if (isRectangleDimensionsAreCorrect(width1, height1, width2, height2)){
```

Comment:
I suggest renaming isRectangleDimensionsAreCorrect to validateDimensions and use exception flow in this method.
Also, I suggest using validation into calculatSquareOfRectangle method.